src/legacy/core_plugins/metrics/public/components/vis_editor_visualization.js

```diff
@@ -171,6 +206,21 @@ class VisEditorVisualization extends Component {
           />
         </EuiFlexItem>
 
+        {formattedPavelIntervalValue &&
+        <EuiFlexItem grow={false}>
+          <EuiText color="default" size="xs">
+            <FormattedMessage
```

Comment:
Can we add a surrounding `<p>` tag as is currently done in the message next to this one? There will no visible changes if in the future EUI will change paddings/margins of the p element inside EuiTexts